queries/vue/injections.scm

```diff
@@ -49,6 +49,18 @@
   (#eq? @_ts "ts")
 )
 
+; <script lang="tsx">
+; <script lang="jsx">
+(
+  (script_element
+    (start_tag
+      (attribute
+        (attribute_name) @_attr
+        (quoted_attribute_value (attribute_value) @language)))
+    (raw_text) @content)
+    (#eq? @_attr "lang")
+    (#any-of? @language "tsx" "jsx"))
+
```

Comment:
I'm nitpicking, but the comment is unnecessary since the code is self documenting, and if you can move up script_element to the first paren like so:
```scm
((script_element
```

That'd be more consistent with the ending format